sdk/deepbook-v3/test/helper/constants.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+export class Constants {
+	public static readonly GAS_BUDGET = 6_000_000_000;
```

Comment:
Is this arbitrary for now and will be changed later?

Reply:
This is the max gas budget to make sure all deployments and tests don't run out of gas.